js/controller.js

```diff
@@ -286,6 +287,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb = null;
     },
 
+    onEmbedCodeChangedAfterOoyalaAd: function(event, embedCode, options) {
+      if (options) {
```

Comment:
Added the code similar to the one that is already existing in onEmbedCodeChanged ()